File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session2/p3.c
#include<stdio.h>
void main(){

	int num=1;

	int row;


	printf("Enter row:\n");
	scanf("%d",&row);
	
	

	for(int i=1;i<=row;i++){
		for(int j=1;j<=row;j++){
			printf("%d ",num);
			num++;
			num++;
			num++;
			
		}
	
		printf("\n");
	}
}

File: java-main/java-main/JavaDSA_Super-X/Assi_3/p3.java
import java.util.Scanner;

class Demo {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int x = scanner.nextInt();

        if (isPalindrome(x)) {
            System.out.println( x + " is a palindrome number.");
        } else {
            System.out.println(x + " is not a palindrome number.");
        }

        scanner.close();
    }
  
  static boolean isPalindrome(int num) {
        int originalNum = num;
        int reversedNum = 0;

        while (num > 0) {
            int remainder = num % 10;
            reversedNum = reversedNum * 10 + remainder;
            num = num / 10;
        }

        return originalNum == reversedNum;
    }
}

